web-app/src/contexts/InfographicsContext.js

```diff
@@ -0,0 +1,38 @@
+import PropTypes from "prop-types";
+import { createContext, useEffect, useState } from "react";
+import { useQuery } from "react-query";
+
+export const InfographicsContext = createContext();
+function InfographicsContextProvider({ children }) {
+	const [count, setCount] = useState("");
+	const { isLoading, error, data } = useQuery("languageData", () =>
```

Comment:
This is not the url to be uesd, but will update the correct one myself, the data is showing in the 2 columns as required, will correct the rows later on.